core/src/main/java/dk/alexandra/fresco/framework/sce/evaluator/BatchedStrategy.java

```diff
@@ -116,4 +108,4 @@
       sceNetworkSupplier.nextRound();
     }
   }
-}
\ No newline at end of file
+}
```

Comment:
Google code style - I have (now had, since this apparently causes too many issues) a trigger within eclipse that formatted the file upon save. This is why some files are touched, but not changed beside formatting.

Reply:
Well, it can hardly be google style to blame.

I would expect a cleanup of this PR to remove the clutter.